supervisor/dbus/network/interface.py

```diff
@@ -33,7 +32,7 @@ class NetworkInterface(DBusInterfaceProxy):
     properties_interface: str = DBUS_IFACE_DEVICE
     sync_properties: bool = False
 
-    def __init__(self, nm_dbus: DBus, object_path: str) -> None:
+    def __init__(self, object_path: str) -> None:
```

Comment:
I have no idea why init was asking for `nm_dbus` here, it did not use it at all. So I removed it as it complicated tests (and any place instances of this class are created).